src/components/map/MapDirective.js

```diff
@@ -80,6 +81,40 @@ goog.require('ga_permalink');
 
             map.setTarget(element[0]);
 
+            scope.$watch('::ol3d', function(ol3d) {
+              if (ol3d) {
+                var camera = ol3d.getCesiumScene().camera;
+                var params = gaPermalink.getParams();
+
+                // initial location based on query params
+                var position = undefined;
+                if (params.lon && params.lat && params.elevation) {
+                  var lon = parseFloat(params.lon);
+                  var lat = parseFloat(params.lat);
+                  var elevation = parseFloat(params.elevation);
+                  position = new Cesium.Cartographic(lon, lat, elevation);
+                }
+                camera.setView({
+                  positionCartographic: position,
+                  heading: params.heading ? parseFloat(params.heading) : undefined,
+                  pitch: params.pitch ? parseFloat(params.pitch) : undefined,
+                  roll: 0.0
+                });
+
+                // update permalink
+                camera.moveEnd.addEventListener(function() {
```

Comment:
> I really would like the have debouncing here to be consistent and on the save side.

See https://github.com/geoadmin/mf-geoadmin3/commit/d06ff54f03f6f45e2b12651155aee67edf916f74

> Side note: is it worth to remove the listener when switching back the 2d?

It's not needed because the camera is not changed in 2d